ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +143,79 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+          computed: '_setBuckets(buckets)',
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _showAll: {
+          type: Boolean,
+          value: false,
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
```

Comment:
Would we want these ones to be also `readOnly`?
I think it is something we started discussing on migration guidelines and it should be applied also to improvements and iterations.